src/components/shared/TractionForm.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <flagged-feature name="enable_custom_form">
+    <template #disabled
+      ><b-form :class="classes" @submit.prevent="submitAction" @reset.prevent="resetAction">
+        <slot />
+      </b-form>
+    </template>
+    <form
+      v-bind="$attrs"
+      :class="classes"
+      action=""
```

Comment:
Can action/reset be removed since they are empty props or are they needed to override default behaviour

Reply:
Yes, that was my understanding that it is is needed to override the default behaviour.